yivi-css/src/examples/helpers.scss

```diff
@@ -16,19 +16,19 @@
 //
 // Trigger helper:
 // <ul>
-//   <li><a href="javascript:eachDo(document.querySelectorAll('.irma-web-header'), function(h) {h.classList.add('irma-web-show-helper')})">Show helpers</a></li>
-//   <li><a href="javascript:eachDo(document.querySelectorAll('.irma-web-header'), function(h) {h.classList.remove('irma-web-show-helper')})">Hide helpers</a></li>
+//   <li><a href="javascript:eachDo(document.querySelectorAll('.yivi-web-header'), function(h) {h.classList.add('yivi-web-show-helper')})">Show helpers</a></li>
+//   <li><a href="javascript:eachDo(document.querySelectorAll('.yivi-web-header'), function(h) {h.classList.remove('yivi-web-show-helper')})">Hide helpers</a></li>
 // </ul>
 //
 // Markup:
-// <section class="irma-web-form">
-//   <header class="irma-web-header irma-web-show-helper">
-//     <p>Ga verder met <i class="irma-web-logo">IRMA</i></p>
-//     <section class="irma-web-helper">
-//       <p>Kom je er niet uit? Kijk dan eerst eens op <a href="https://irma.app/">de website van IRMA</a>.</p>
+// <section class="yivi-web-form">
+//   <header class="yivi-web-header yivi-web-show-helper">
+//     <p>Ga verder met <i class="yivi-web-logo">Yivi</i></p>
+//     <section class="yivi-web-helper">
+//       <p>Kom je er niet uit? Kijk dan eerst eens op <a href="https://yivi.app/">de website van Yivi</a>.</p>
 //     </section>
 //   </header>
-//   <section class="irma-web-content">
+//   <section class="yivi-web-content">
```

Comment:
These colors look a bit weird. I think the $helper-color needs to be different.
![image](https://user-images.githubusercontent.com/17035191/189610794-4f09119b-3691-459d-a974-3a72112ed7f9.png)
Maybe it would be good to let Rosanne do a design check.

Reply:
She checked the revised version, and approves